project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -1914,17 +1932,26 @@ record MeasurementRegistrationInformationPxP(
       String msDeviceCURIE,
       String samplePoolGroup,
       String facility,
-      String fractionName,
       String digestionEnzyme,
       String digestionMethod,
       String enrichmentMethod,
       String injectionVolume,
       String lcColumn,
       String lcmsMethod,
       String labelingType,
-      HashMap<String, MeasurementSpecificPxP> specificMetadata
+      Map<String, MeasurementSpecificPxP> specificMetadata
   ) implements ValidationRequestBody {
 
+    /**
+     * Returns the {@link List} of sample identifiers this measurement refers to.
+     *
+     * @return the {@link List} of sample identifiers
+     * @since 1.10.0
+     */
+    public List<String> referencedSamples() {
```

Comment:
```suggestion
    public List<String> measuredSamples() {
```
Is this the same as measured samples? referenced samples sounds very technical